src/PKSim.Core/Snapshots/OriginData.cs

```diff
@@ -43,17 +44,15 @@ public class OriginData
       /// </summary>
       public Parameter Height { get; set; }
 
-
       public ValueOrigin ValueOrigin { get; set; }
 
-      /// <summary>
-      /// Name of disease state associated with OriginData. Null if no disease state defined
-      /// </summary>
+      //Null if no disease state defined
+      public DiseaseState Disease { get; set; }
```

Comment:
Bevcause we had DiseaseStateProperty already, I had to name it differently so that we can support older snapshot versions